packages/contract-wrappers/src/contract_wrappers/erc20_token_wrapper.ts

```diff
@@ -435,13 +435,15 @@ export class ERC20TokenWrapper extends ContractWrapper {
         if (!_.isUndefined(tokenContract)) {
             return tokenContract;
         }
-        const [abi, address] = await this._getContractAbiAndAddressFromArtifactsAsync(
-            artifacts.ERC20Token,
-            normalizedTokenAddress,
-        );
+        // TODO(albrow): Do we really still need this check? The default error
```

Comment:
@LogvinovLeon or @fabioberger thoughts on this TODO? (Also appears a few other places).

Reply:
What is the default error? `doesContractExistAtAddressAsync` returns a boolean.

Reply:
What I'm asking is whether we really need the `ContractWrappersError.ERC20TokenContractDoesNotExist` error. If we remove this check, `Web3Wrapper` throws an error along the lines of `"Tried to send a transaction to 0xabcde... but it is not a contract address."`. Together with a stack trace, I think this makes it pretty clear what went wrong. What do you think?

Reply:
Yep, let's get rid of the mapping.